src/jit/codegencommon.cpp

```diff
@@ -9406,9 +9422,6 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
             compiler->unwindBegEpilog();
             unwindStarted = true;
         }
-        // mov FP into SP
-        inst_RV_RV(INS_mov, REG_SPBASE, REG_FPBASE);
-        compiler->unwindSetFrameReg(REG_FPBASE, 0);
```

Comment:
If you're going to remove the instruction, you can remove this whole clause. There's no need to call unwindBegEpilog() if we're not generating any unwind codes (or instructions) here.

---
In reply to: [57769409](https://github.com/dotnet/coreclr/pull/3966#discussion_r57769409) [](ancestors = 57769409,57753045)